src/main/java/FileWriterClass.java

```diff
@@ -0,0 +1,99 @@
+
+import java.io.*;
```

Comment:
It's recommended to list the imported classes explicity rather than using the wildcard